src/components/ChannelList/ChannelList.tsx

```diff
@@ -75,6 +75,7 @@ export type ChannelListProps<
     channel: Channel<StreamChatGenerics>,
     t: TranslationContextValue['t'],
     userLanguage: TranslationContextValue['userLanguage'],
+    isMessageAIGenerated?: ChatContextValue['isMessageAIGenerated'],
```

Comment:
Had to add this as an optional argument to make sure the change isn't breaking, as `getLatestMessagePreview` is overridable in the SDK.